From 15ca023d767adb1d9528decfaa5383f7e340e672 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Wed, 12 Jul 2017 11:04:41 +0800
Subject: [PATCH 2112/5242] MLK-15955 gpu: imx: dpu: Sort dpu unit names in
 struct dpu_soc alphabetically

commit  00dbdb8392a15b256b153fb3d787bcfeebe4f6f6 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch sorts DPU unit names in structure dpu_soc alphabetically.
So, just code change only.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu/dpu-prv.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/imx/dpu/dpu-prv.h b/drivers/gpu/imx/dpu/dpu-prv.h
index ae58468..caf95d1 100644
--- a/drivers/gpu/imx/dpu/dpu-prv.h
+++ b/drivers/gpu/imx/dpu/dpu-prv.h
@@ -176,8 +176,8 @@ struct dpu_soc {
 	struct dpu_disengcfg	*dec_priv[2];
 	struct dpu_extdst	*ed_priv[4];
 	struct dpu_fetchdecode	*fd_priv[4];
-	struct dpu_fetchlayer	*fl_priv[2];
 	struct dpu_framegen	*fg_priv[2];
+	struct dpu_fetchlayer	*fl_priv[2];
 	struct dpu_hscaler	*hs_priv[3];
 	struct dpu_layerblend	*lb_priv[7];
 	struct dpu_tcon		*tcon_priv[2];
-- 
1.7.9.5

